### DISCUSSION TITLE
v4.0.0-beta2

### DISCUSSION THREAD

**Discussion by danny007in** [ColorlibHQ/AdminLTE#5681] [General]
## BREAKING CHANGES
* Removed `.layout-fixed-complete` class and component.

## What's Changed
* Update push-menu.ts by @burdittw in https://github.com/ColorlibHQ/AdminLTE/pull/5583
* Added Icons in UI by @Developermayank in https://github.com/ColorlibHQ/AdminLTE/pull/5652

## Minor
* Bug Fixes

## New Contributors
* @Developermayank made their first contribution in https://github.com/ColorlibHQ/AdminLTE/pull/5652

**Full Changelog**: https://github.com/ColorlibHQ/AdminLTE/compare/v4.0.0-beta1...v4.0.0-beta2

<hr /><em>This discussion was created from the release <a href='https://github.com/ColorlibHQ/AdminLTE/releases/tag/v4.0.0-beta2'>v4.0.0-beta2</a>.</em>

**Comment by dereuromark** [comment]
Is there any chance that `layout-navbar-fixed` and a fixed top navbar remains?
Coming from v3 this seems to be lost now.

**Reply by danny007in** [reply]
upcoming version i will add, please wait

**Comment by AhmedAlghoul** [comment]
Hi 
I can't find the starter page in adminlte-v4.0.0-beta2-with-dist.
